docs/components/toast/README.md

```diff
@@ -0,0 +1,248 @@
+    <cdr-toast
+      :open="opened"
+      type="success"
+      @closed="opened = false">
+      <template #icon-left>
+        <icon-check-fill inherit-color>
+      </template>
+      I am a success toast
+    </cdr-toast>
+  </div>
+</div>
+```
+</cdr-doc-example-code-pair>
+
+## Persistent toast
+
+CdrToast provides an optional `autoDismiss` property to disable the auto-dismiss functionality.
+
+<cdr-doc-example-code-pair repository-href="/src/components/CdrToast"
+:load-sprite="true"
+:sandbox-data="Object.assign({}, $page.frontmatter.sandboxData, {components: 'CdrToast, CdrChip, IconWarningFill'})" :model="{ opened: false }" >
+
+```html
+<div>
+  <cdr-chip
+    :aria-pressed="opened"
+    @click="opened = true"
+  >
+    Trigger toast
+  </cdr-chip>
+  <!-- CdrToast should always be contained within an absolutely 
+  positioned element in the top right of your page -->
+  <div style="position: absolute; top: 2rem; right: 1.5rem">
+    <cdr-toast
+      :open="opened"
+      :auto-dismiss="false"
+      type="warning"
+      @closed="opened = false">
+      <template #icon-left>
+        <icon-warning-fill inherit-color>
+      </template>
+      I am a warning toast and must be dismissed by user action
+    </cdr-toast>
+  </div>
+</div>
+```
+</cdr-doc-example-code-pair>
+
+## Accessibility
+
+Many WCAG requirements are contextual to their implementation. To ensure that usage of this component complies with accessibility guidelines you are responsible for the following:
+- Add `role="alert"` for messages that are important or time sensitive to ensure that the messaging is immediately announced to screen readers. For other content, add `aria-live="polite"`.
+- Don't rely on color alone to convey your message. Provide an additional indicator to color, like an icon: [WCAG 1.4.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html)
+- Error Identification techniques and criteria: [WCAG 3.3.1](https://www.w3.org/WAI/WCAG21/Understanding/error-identification.html)
+
+# Guidelines
+
+There are five information types supported for toasts: **error, warning, success, informational, and default**. Each type corresponds with a color and icon to provide a consistent, universal experience for users.
+
+### **Error**
+Use to inform that something went wrong. They affect or block the user's experience and must be resolved before moving forward.
+
+### **Warning**
+Use for a message requiring attention but not resolution in order to continue. Warning toasts might tell a user what could happen if they don’t address what they’re being warned about.
+
+### **Success**
+Use to communicate that an action has been successfully completed. Provides a positive response to user actions. No action is required.
+
+### **Informational**
+Use to provide context around a situation. No action is required.
+
+### **Default**
+Use to provide generic messaging that does not fit the other types
+
+## Use when
+
```

Comment:
/patterns/status-notifications/#use-when-2 - these should be the same - or mostly the same 
maybe update both to ensure all use-cases are covered?